src/System.Net.Sockets/src/System/Net/Sockets/SocketAsyncEventArgs.Windows.cs

```diff
@@ -556,11 +556,11 @@ private void InnerStartOperationSendPackets()
             {
```

Comment:
Above this code, `m_SendPacketsElementsFileCount` and `m_SendPacketsElementsBufferCount` have not been updated to the `_sendPacketsElementsBufferCount`.